docs/docs.jsx

```diff
@@ -23,5 +23,9 @@ class Docs extends React.Component {
   }
 }
 
-const content = document.getElementById("content");
-ReactDOM.render(<Docs/>, content);
+if (typeof document !== "undefined") {
```

Comment:
Since `static-site-generator-webpack-plugin` parses/executes the code in a Node context while building its HTML, we need to add this check. :(

The better pattern might be move the `ReactDOM.render` stuff out of `docs` altogether and just export the class. Then we could have an `app.jsx` or `entry.jsx` or something that requires it in and renders it like normal, plus a `static-render-entry.jsx` that 1.) performs `document` check & renders app and 2.) does its current work of rendering HTML. That way, people *not* working in `static-render-entry` don't even have to know about the plugin.